client/ipa-client-install

```diff
@@ -1569,8 +1569,9 @@ def get_local_ipaddresses(iface=None):
             for ip in if_addrs.get(family, []):
                 try:
                     ips.append(ipautil.CheckedIPAddress(ip['addr']))
-                except ValueError:
-                    continue
+                    root_logger.debug('IP check sucessfull: %s' % ip['addr'])
+                except ValueError as e:
+                    root_logger.debug('IP check failed: %s' % str(e))
```

Comment:
It might be useful to log success as well. I decided to implement it, because it was suggested in the ticket. Will fix the typo.